src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -98,6 +114,15 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
             list($alias, $field) = $this->addJoinsForNestedProperty($property, $alias, $queryBuilder, $queryNameGenerator);
         }
 
+        if (null !== $nullsComparison = $this->properties[$property]['nulls_comparison'] ?? null) {
+            $nullsDirection = (self::NULLS_SMALLEST === $nullsComparison) ? $direction : ('ASC' === $direction ? 'DESC' : 'ASC');
+
+            $nullRankHiddenField = sprintf('_%s_%s_null_rank', $alias, $field);
+
+            $queryBuilder->addSelect(sprintf('CASE WHEN %s.%s IS NULL THEN 0 ELSE 1 END AS HIDDEN %s', $alias, $field, $nullRankHiddenField));
```

Comment:
HIDDEN is a DQL keyword that hides the selected field from the result set.

http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/dql-doctrine-query-language.html